src/commands.ts

```diff
@@ -11,6 +11,8 @@ import { SSHSession, SSHChannel } from './ssh_session'
 import { Failure } from './session'
 import { NativeBiometric } from 'capacitor-native-biometric'
 import { Capacitor } from "@capacitor/core"
+import { Filesystem, Directory , Encoding} from '@capacitor/filesystem'
```

Comment:
here to there's an extra new line after from

Reply:
what do you mean by extra new line , i dont see it in my code